src/Nights.jsx

```diff
@@ -0,0 +1,8 @@
+import moment from "moment";
+
+export const numberOfNight = ({ checkInDate, checkOutDate }) => {
+  const checkIn = moment(checkInDate);
+  const checkOut = moment(checkOutDate);
+  const nights = checkOut.diff(checkIn, "days");
+  return nights;
+};
```

Comment:
Because you're not returning any `jsx` (this isn't a react component), this should just be a regular `.js` file instead of a `.jsx` file